scripts/archived_scripts/community_scripts/amm_trade_example.py

```diff
@@ -12,7 +12,7 @@ class AmmTradeExample(ScriptStrategyBase):
     This example shows how to call the /amm/trade Gateway endpoint to execute a swap transaction
     """
     # swap params
-    connector_chain_network = "uniswap_ethereum_goerli"
+    connector_chain_network = "amm_ethereum_goerli"
```

Comment:
We have connector / exchange named uniswap
https://hummingbot.org/exchanges/uniswap/

this changes here not needed